src/main/java/frc/robot/Robot.java

```diff
@@ -67,6 +74,8 @@ public void robotInit() {
     @Override
     public void robotPeriodic() {
         CommandScheduler.getInstance().run();
+
+        motor.setControl(new DutyCycleOut(1));
```

Comment:
From the [Phoenix 6 documentation](https://pro.docs.ctr-electronics.com/en/latest/docs/api-reference/api-usage/control-requests.html#modifying-a-control-request):
> Java users should reuse control requests to prevent excessive invocation of the Garbage Collector.

You can create the request object as a class member and change the value as needed with [`withPosition()`](https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/controls/PositionVoltage.html#withPosition(double)) (this doesn't create a new object but changes the same one).

Reply:
I'll change it.

Reply:
Ah hold on, I deleted this anyway.